cookbook/core/control_flow/chain_tasks.py

```diff
@@ -57,7 +52,6 @@ def chain_tasks_wf() -> pd.DataFrame:
 
```

Comment:
Can you modify the above code to

```python
write_node= write()
read_node = read()

write_node >> read_node
```

Reply:
Can you also verify if the code works with the latest flytekit version — locally and on sandbox?